src/browser/ui/__tests__/ReactMount-test.js

```diff
@@ -144,4 +144,13 @@ describe('ReactMount', function() {
     ReactMount.render(<div />, container);
     expect(console.warn.mock.calls.length).toBe(0);
   });
+
+  it('should warn when mounting into document.body', function () {
+    spyOn(console, 'warn')
+    ReactMount.render(<div />, document.body);
```

Comment:
Aha, I was running command line via `npm test` /  jest. I'll take fix this, thanks for the tip!